### ISSUE TITLE
Suggestion - Analog gearshift / adjustable clutch

### ISSUE THREAD

**Issue by grifdail** [hlysine/create_connected#84]
Right now the only way to change the rotation speed of a machine based on a redstone signal level are through the use of adjustable chain gearshift but these are only limited to 1 to 2 range.

My suggestion would be the addition of a block that divide the input rotation speed by the level of the redstone power.

For instance, the gearshift receive an input of 255rmp. At 0 redstone it output at 255rmp, at 8 redstone, it output a 128rpm, at 13 it output 34rpm and at 15 it act like a clutch and output 0 rpm.

The formula would be `output_speed = input_speed - round((redstone_level / 15) * input_speed )`

Of course an inverted version that only transmit rotation when powered is also possible.

Even though it shouldn't break the game balancing, as it can only output rotation speed below the input one, i still thing it would be better if the recipe is relatively late game (Involving brass maybe ?)

**Comment by DJCatalyst53** [comment]
Clockwork adds a block called the Resistor that does the sort of what you are looking for. It takes one-fifteenth of the rotational force per strength of redstone.

**Comment by TheBigSTN** [comment]
@grifdail 
I don't think it uses that formula but Create aeronautics added analog transmission.

**Comment by hlysine** [comment]
With adjustable chain gearshift and analog transmission, I don't think we need another block to scale RPM with redstone anymore
